src/nvim/log.h

```diff
@@ -67,6 +67,30 @@
        __VA_ARGS__)
 #  endif
 
+# if __linux__
+#  include <execinfo.h>
+#  define LOG_CALLSTACK(prefix) \
+  do { \
+    void *trace[100]; \
+    int trace_size = backtrace(trace, 100); \
+    \
+    char exe[1024]; \
```

Comment:
Should not it be MAXPATHL? Or do not do `readlink` at all, construct `/proc/{pid}/exe`.